Console.CodingTracker.mefdev/Data/CodingSessionRepository.cs

```diff
@@ -0,0 +1,155 @@
+using System.Transactions;
+using CodingLogger.Models;
+using Dapper;
+using Microsoft.Data.Sqlite;
+using Spectre.Console;
+
+namespace CodingLogger.Data
+{
+    public class CodingSessionRepository
+    {
+        public string _connectionString;
+
+        public CodingSessionRepository(string connectionString)
+        {
+            _connectionString = connectionString;
+        }
+        private SqliteConnection GetConnection()
+        {
+            var connection = new SqliteConnection(_connectionString);
+            connection.Open();
+            return connection;
+        }
+
+        public async Task Create(CodingSession codingSession)
+        {
+            try
+            {
+                using (var connection = GetConnection())
```

Comment:
Remember that you can use the simpler "Using Declaration" syntax to save an indentation level against the "Using Statement" syntax:
```
// Using Statement
using (var connection = new SqliteConnection(connectionString))
{
    // ...
}

// vs

// Using Declaration
using var connection = new SqliteConnection(connectionString);
// ...
```